cmd/node/config/enableEpochs.toml

```diff
@@ -248,8 +248,7 @@
         { EnableEpoch = 1, Type = "KOSK"}
     ]
 
-    # GuardAccountFeatureEnableEpoch represents the epoch when the guard account feature is enabled in the protocol
-    GuardAccountFeatureEnableEpoch = 2
+    # SetGuardianEnableEpoch represents the epoch when the guard account feature is enabled in the protocol
```

Comment:
`MetricGuardAccountFeatureEnableEpoch` should be removed(not used) and perhaps add a new one for `SetGuardianEnableEpoch`